src/components/NotificationsDrawer/DrawerPanel.tsx

```diff
@@ -0,0 +1,515 @@
+            </StackItem>
+            <StackItem className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
+              <Link
+                onClick={onLinkClick}
+                to="/settings/notifications/user-preferences"
+              >
+                Check your Notification Preferences
+              </Link>
+            </StackItem>
+            <StackItem className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
+              <Link
+                onClick={onLinkClick}
+                to="/settings/notifications/notificationslog"
+              >
+                View the Event log to see all fired events
+              </Link>
+            </StackItem>
+            <StackItem className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
+              <Text>Contact your organization administrator</Text>
+            </StackItem>
+          </Stack>
+        </>
+      )}
+    </EmptyStateBody>
+  </EmptyState>
+);
+
+const DrawerPanelBase = ({
+  isOrgAdmin,
+  getUserPermissions,
+  toggleDrawer,
+}: DrawerPanelProps) => {
+  const { addWsEventListener } = useChrome();
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [isFilterDropdownOpen, setIsFilterDropdownOpen] = useState(false);
+  const [activeFilters, setActiveFilters] = useAtom(
+    notificationDrawerFilterAtom
+  );
+  const navigate = useNavigate();
+  const populateNotifications = useSetAtom(notificationDrawerDataAtom);
+  const addNotification = useSetAtom(addNotificationAtom);
+  const notifications = useAtomValue(notificationDrawerDataAtom);
+  const selectedNotifications = useAtomValue(notificationDrawerSelectedAtom);
+  const updateSelectedNotification = useSetAtom(updateNotificationSelectedAtom);
+  const [hasNotificationsPermissions, setHasNotificationsPermissions] =
+    useState(false);
+  const updateNotificationRead = useSetAtom(updateNotificationReadAtom);
+  const updateAllNotificationsSelected = useSetAtom(
+    updateNotificationsSelectedAtom
+  );
+  const [filterConfig, setFilterConfig] = useState<FilterConfigItem[]>([]);
+  const eventType: ChromeWsEventTypes =
+    'com.redhat.console.notifications.drawer';
+  const handleWsEvent = (event: ChromeWsPayload<NotificationsPayload>) => {
+    console.log('new event');
+    addNotification(event.data as unknown as NotificationData);
+  };
+
+  useEffect(() => {
+    let mounted = true;
+    const unregister = addWsEventListener(eventType, handleWsEvent);
+    const getNotifications = async () => {
+      try {
+        const { data } = await axios.get<{ data: NotificationData[] }>(
+          `/api/notifications/v1/notifications/drawer`,
+          {
+            params: {
+              limit: 50,
+              sort_by: 'read:asc',
+              startDate: getSevenDaysAgo(),
+            },
+          }
+        );
+        populateNotifications(data?.data || []);
+      } catch (error) {
+        console.error('Unable to get Notifications ', error);
+      }
```

Comment:
We have the JS client for notifications installed. Let's update the `notificationsApi` to include drawer endpoint and use it here instead of axios get.